### PR TITLE
Shows near & far neighbors

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

**Comment by pkubanek** [comment]
Uploading Záznam obrazovky 2021-08-16 v 1.14.30 odp..mov…

See screencast documenting GUI/EUI functionality & simulator capable to support mirror with disabled FAs.

**Comment by pkubanek** [comment]
Uploading Záznam obrazovky 2021-08-16 v 1.14.30 odp..mov…

Also shows near & far neighbors in GUI, better error reporting, refactor GUI/EUI to support charting variables (next task..) - raising to 3 SP.

File: python/lsst/ts/cRIOpy/GUI/ActuatorsDisplay/ForceActuator.py [hunk 1]
```diff
@@ -109,9 +117,9 @@ def updateData(self, data, state):
             self._state = state
             self.update()
 
-    def setSelected(self, selected):
-        """Set actuator selection status."""
-        self._selected = selected
+    def setKind(self, kind):
```
Comment (teweitsai) [inline python/lsst/ts/cRIOpy/GUI/ActuatorsDisplay/ForceActuator.py:120]: Please consider the doc string of `kind` in Parameters.

File: python/lsst/ts/cRIOpy/GUI/ActuatorsDisplay/ForceActuator.py [hunk 2]
```diff
@@ -148,12 +156,23 @@ def boundingRect(self):
             20 * self._scale_factor,
         )
 
+    def getValue(self):
+        """Returns current value, string formated to scale."""
+        return self.formatValue(self.data)
+
+    def formatValue(self, v):
```
Comment (teweitsai) [inline python/lsst/ts/cRIOpy/GUI/ActuatorsDisplay/ForceActuator.py:163]: Please consider the doc string.

File: python/lsst/ts/cRIOpy/GUI/EnabledForceActuators.py [hunk 3]
```diff
@@ -194,22 +194,34 @@ def detailedState(self, data):
     @Slot(map)
     def enabledForceActuators(self, data):
         """Callback with enabled FA data. Triggers display update."""
-        self.mirrorWidget.mirrorView.clear()
+        if len(self.mirrorWidget.mirrorView.items()) == 0:
+            new = True
```
Comment (teweitsai) [inline python/lsst/ts/cRIOpy/GUI/EnabledForceActuators.py:198]: What is `new`?

File: python/lsst/ts/cRIOpy/GUI/ForceActuatorGraphPageWidget.py [hunk 4]
```diff
@@ -24,16 +24,17 @@ def __init__(self, m1m3):
             self.updateSelectedActuator
         )
 
-    def updateValues(self, data):
+    def updateValues(self, data, changed):
```
Comment (teweitsai) [inline python/lsst/ts/cRIOpy/GUI/ForceActuatorGraphPageWidget.py:27]: What is `changed`?

**Review by teweitsai — APPROVED** [review]
The PR looks good to me. Please consider to improve the doc string of functions and add the unit tests if possible. Thanks!

**Pull request merged**